.github/workflows/flaskproject.yaml

```diff
@@ -86,10 +80,18 @@ jobs:
 
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v1
+        with:
+          output: flaskproject/flaskproject-codeql.sarif
```

Comment:
Also feel like `work-directory` doesnt work here since I checked out how codeql's analyze was written (https://github.com/github/codeql-action/blob/main/analyze/action.yml) so preemptively making it abs path